### ISSUE TITLE
webfont not working now 'Access-Control-Allow-Origin'

### ISSUE THREAD

**Issue by Mnilionic** [ionic-team/ionicons#737]
console log:
Access to font at 'https://unpkg.com/ionicons@4.5.10-0/dist/fonts/ionicons.woff2?v=4.5.9-1' from origin 'http://devi4nik.club' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'https://filmbaz.co' that is not equal to the supplied origin.
jquery.min.js?v=1.10.2:4 GET https://unpkg.com/ionicons@4.5.10-0/dist/fonts/ionicons.woff2?v=4.5.9-1 net::ERR_FAILED
(anonymous) @ jquery.min.js?v=1.10.2:4
c @ jquery.min.js?v=1.10.2:4
fireWith @ jquery.min.js?v=1.10.2:4
ready @ jquery.min.js?v=1.10.2:4
q @ jquery.min.js?v=1.10.2:4
(index):1 Access to font at 'https://unpkg.com/ionicons@4.5.10-0/dist/fonts/ionicons.woff?v=4.5.9-1' from origin 'http://devi4nik.club' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'https://filmbaz.co' that is not equal to the supplied origin.
(index):1 GET https://unpkg.com/ionicons@4.5.10-0/dist/fonts/ionicons.woff?v=4.5.9-1 net::ERR_FAILED
(index):1 Access to font at 'https://unpkg.com/ionicons@4.5.10-0/dist/fonts/ionicons.ttf?v=4.5.9-1' from origin 'http://devi4nik.club' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'http://hen-party.loc' that is not equal to the supplied origin.
(index):1 GET https://unpkg.com/ionicons@4.5.10-0/dist/fonts/ionicons.ttf?v=4.5.9-1 net::ERR_FAILED

**Comment by balbaal** [comment]
i've same issue blocked by CORS. to solved this i'm trying to downgrade the version, and using another host ionicons from CDN : https://cdnjs.com/libraries/ionicons/4.5.5.
just for a while !!

**Comment by miktam** [comment]
Replacing 

`https://unpkg.com/ionicons@4.5.10-0/dist/css/ionicons.min.css`

with 

`https://cdnjs.cloudflare.com/ajax/libs/ionicons/4.5.5/css/ionicons.min.css`

works for me.

**Comment by DonnyR1991** [comment]
Same problem here, it looks like the cdn has references to random webshops...

**Comment by rawteech** [comment]
> Replacing
> 
> `https://unpkg.com/ionicons@4.5.10-0/dist/css/ionicons.min.css`
> 
> with
> 
> `https://cdnjs.cloudflare.com/ajax/libs/ionicons/4.5.5/css/ionicons.min.css`
> 
> works for me.

Is the cdn official? Because the one provided by [ionicons](https://ionicons.com/usage/) is `https://unpkg.com/ionicons@4.5.10-0/dist/css/ionicons.min.css`

**Comment by gbaggaley** [comment]
> Is the cdn official? Because the one provided by [ionicons](https://ionicons.com/usage/) is `https://unpkg.com/ionicons@4.5.10-0/dist/css/ionicons.min.css`

Might not be official but cdnjs is trustworthy. I changed it to cdnjs and it's working again now, could just be an issue with unpkg as it's using the same ionicons version.

**Comment by Mnilionic** [comment]
unpkg.com recovered. Looks like they had Halloween

**Comment by webinpixels** [comment]
unpkg is buggy - too many down times, I change to https://cdnjs.cloudflare.com/ajax/libs/ionicons/4.5.6/ionicons.js - seems more stable

**Comment by amandaejohnston** [comment]
It sounds like this was already resolved, so I'll go ahead and close it out. If you're still having problems in the latest version of Ionicons, please open a new issue and include a minimal reproduction. Thank you!
